src/utils/trpc.ts

```diff
@@ -1,6 +1,7 @@
 import { httpBatchLink } from "@trpc/client";
 import { createTRPCNext } from "@trpc/next";
 import type { AppRouter } from "@server/routers/_app";
+import Superjson from "superjson";
```

Comment:
this one's uppercase but the other one is lowercase :(

stupid default exports